subsys/bluetooth/crypto/bt_crypto.h

```diff
@@ -2,6 +2,9 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#ifndef __BT_CRYPTO_H
+#define __BT_CRYPTO_H
```

Comment:
Is this necessary? Typically we don't do include guards in internal header files

Reply:
I got asked the same question by @theob-pro. It's a mixed bag: some internal header files do it, some don't. In this case, given that this will be included by multiple parts of the Bluetooth subsystem (mesh, host, audio...) I think it makes sense.

Reply:
Yep, makes sense to me too.

Reply:
All headers must be guarded. How are we going to create self-contained headers if, e.g. A includes B and C (it depends on both), and B includes C?

Reply:
yes I also got different answers before. @carlescufi It would be nice if we could have an official guideline for the whole project instead of people's disparate opinions.